store/src/main/java/com/nytimes/android/external/store/base/beta/Store.java

```diff
@@ -63,12 +64,21 @@
     /**
      * Clear the memory cache of all entries
      */
+    @Deprecated
     void clearMemory();
 
     /**
      * Purge a particular entry from memory cache.
      */
+    @Deprecated
     void clearMemory(@Nonnull V key);
 
+    void clear();
```

Comment:
Add a comment so that users will know how clear works. Clears memory, but also clears disk when persister implements clearable.